modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

```diff
@@ -0,0 +1,272 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-higher-vm-workload-density.adoc.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-using-wasp-agent-to-configure-higher-vm-workload-density_{context}"]
+= Using wasp-agent to configure higher VM workload density
+
+`wasp-agent` is a component that enables an OpenShift cluster to assign swap resources to VM workloads. See https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/
```

Comment:
Does `See https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/` relates to this paragraph?